### PR TITLE
Slides for the ad-hoc webinar on components.

### LINKED ISSUE #175: Webinar: Intro to Angular 2 Components

### PR DESCRIPTION
**Pull request by SethDavenport**
Fixes #175

### CODE REVIEW

**Comment by SethDavenport** [comment]
@winkerVSbecks @bertrandk 

Since I'm planning to reuse this for the webinar series, I'd appreciate if one of you could have a look through the slides at some point in the next week or so.

I tried to build up a case for granular component-oriented architecture in Angular2 - similar to how we do React.  It's actually largely based on an Angular2 implementation of the first bit of the atomic design article.

Any feedback would be helpful :)

File: webinar-series/components/slides.md [hunk 1]
```diff
@@ -1,10 +1,761 @@
...
+
+.img-max-width[![right-bar](../../img/bar-max-right.svg)]
+
+.flex.justify-between.bold.rangle-purple.caps[
+  .block[Less Complex]
+  .block[More Complex]]
+
+---
+
+### Smart vs. Dumb Components
```
Comment (winkerVSbecks) [inline webinar-series/components/slides.md]: Have you considered using the terms Presentational & Container components. I've been seeing those everywhere lately – even the ngCourse uses it in the redux section. IMO it sounds better.

**Comment by winkerVSbecks** [comment]
@SethDavenport this is perfect! We should consider re-writing ngCourse component section using this structure. 

One "feature" you could consider adding is defining properties that have 2-way data binding.

File: webinar-series/components/slides.md (hunk 1 above)
Comment (SethDavenport) [inline webinar-series/components/slides.md]: Sure sounds good

**Comment by SethDavenport** [comment]
I touch on two way binding in the input field section but I can expand it a bit

**Pull request merged**